MODULE.bazel

```diff
@@ -28,3 +28,10 @@ register_toolchains(
     "@rules_perl//perl:perl_linux_arm64_toolchain",
     "@rules_perl//perl:perl_windows_x86_64_toolchain",
 )
+
+dev_repos = use_extension("@rules_perl//perl:extensions.bzl", "perl_dev_repositories", dev_dependency = True)
+use_repo(
+    dev_repos,
+    "fcgi",
+    "genhtml",
```

Comment:
In Bzlmod you could also add
```Starlark
bazel_dep(name = "lcov", version = "2.2")
```
and run
```console
❯ bazel run @lcov//:genhtml -- --help
```

Reply:
That was a quick review :joy:, I didn't even check if genhtml was used, I was chasing why perl_xs wasn't working on my rules_cpan fork, let me remove genhtml most likely we don't care at all

Reply:
yea I haven't looked into making xs deps work in rules_cpan. Would be happy to review that addition when you can make it work!

Reply:
We could replace with the lcov import, but it's not so trivial, let me know what you would like to do

Reply:
@skeletonkey we need your advice here, would you prefer me adding a missing dependency or do a major change in the rule to replace with lcov as suggested?

Reply:
@manuelnaranjo Unfortunately, I am not using Bazel regularly now; hence, I'm happy to follow your lead. If you are comfortable making the change, believe that is the better path, and have the time, please go ahead.

Reply:
I think the current setup is cleaner TBH, let's stick with it.